gateway-service-impl/src/test/resources/expected-test-responses/explore/simple-aggregations.json

```diff
@@ -16,7 +16,7 @@
         },
         "RATE#results/rateLatency:Api.Trace|duration": {
           "valueType": "DOUBLE",
-          "double": 4774.983888888889
```

Comment:
why is this change?

Reply:
Got it. But, wasn't it comparing the final value?

Reply:
resolved